tls/Makefile

```diff
@@ -16,4 +16,4 @@
 # this program; if not, write to the Free Software Foundation, Inc., 59
 # Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 
-obj-m = ttls/ programs/
```

Comment:
I believe we don't need the testing programs and can remove `tls/programs` at all.